### PR TITLE
Issue 265: Removes "akeneo-fpm" and merge branch php-7.0 into php-7.1

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

This PR merged the changes done on `php-5.6` and `php-7.0` branches, and removes the `akeneo-fpm` Dockerfile from `php-7.1` branch.

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added tests                       | -
| Changelog updated                 | OK
| Documentation                     | OK
| Fixed tickets                     | #265

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
   [php-5.6 image]: https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-5.6
   [php-5.6]: https://travis-ci.org/akeneo/Dockerfiles/tree/php-5.6
 
 ## Images available
 
 - [**akeneo/php**](php/README.md): Base image with PHP CLI preconfigured, based on `debian:stretch-slim`
-- [**akeneo/fpm**](fpm/README.md): An image with PHP FPM preconfigured to be use with any Symfony project, based on `akeneo/php`
-- [**akeneo/apache-php**](apache-php/README.md): An image with Apache + mod_php preconfigured to be use with any Symfony project, based on `akeneo/php`
-- [**akeneo/akeneo-fpm**](akeneo-fpm/README.md): An image for Akeneo development with PHP FPM, based on `akeneo/fpm`
+- [**akeneo/fpm**](fpm/README.md): An image with PHP FPM preconfigured to be use with any PHP project, based on `akeneo/php`
```
Comment (tamarasaurus) [inline README.md]: "to be used with"

File: README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+- [**akeneo/apache-php**](apache-php/README.md): An image with Apache + mod_php preconfigured to be use with any PHP project, based on `akeneo/php`
```
Comment (tamarasaurus) [inline README.md]: "to be used with"

File: apache-php/README.md [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
+This is a basic Docker environment for PHP development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php). **This image does not contain Akeneo PIM**.
 
-It provides a pre-configured Apache (2.4) + mod_php (PHP 7.1) web server, based on Debian 8 (Jessie).
-Apache is configured to run with the `docker` user. It is then preferred to place your PHP application in `/home/docker`
-and modify the default VHOST to access it.
+It provides a pre-configured Apache (2.4) + mod_php ( PHP 7.1 from [Sury repository](https://deb.sury.org/)) web server, based on Debian 9 (Stretch).
 
-The environment come with  some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug
-(this last one comes deactivated, run `phpenmod xdebug` and restart Apache to enable it).
+The environment come with some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated, run `phpenmod xdebug` and restart Apache to enable it).
```
Comment (tamarasaurus) [inline apache-php/README.md]: "comes with"

File: fpm/README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 
-It comes with PHP 7.1 (provided by dotdeb.org repository), and some PHP extensions: fpm of course,
-but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated,
-run `phpenmod xdebug` to enable it).
+It is intended to be used [httpd](https://hub.docker.com/_/httpd) with FCGI enable or [nginx](https://hub.docker.com/_/nginx).
 
-The PHP FPM daemon is configured with "docker" as user and group, and listen to the socket "/run/php/php7.1-fpm.sock".
+It comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: fpm of course, but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+
+The PHP FPM daemon is configured with "docker" as user and group, and listen to the port `9001`.
```
Comment (tamarasaurus) [inline fpm/README.md]: "listens to"

**Review by tamarasaurus — APPROVED** [review]

File: php/README.md [hunk 5]
```diff
@@ -1,11 +1,36 @@
...
+The environment comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+
+## How to use it?
+
+### From Docker hub
+
+You can directly pull this image from [Docker hub](https://hub.docker.com/r/akeneo/apache-php/) by running:
+
+```bash
+$ docker run -d --name fpm akeneo/php:7.1
```
Comment (ahocquard) [inline php/README.md:16]: `akeneo-fpm` for consistency

File: php/README.md [hunk 6]
```diff
@@ -1,11 +1,36 @@
...
+```
+
+Access the URL `localhost:8080` with your web browser to check that the container works.
+
+### From GitHub
+
+Clone the repository, go inside the created folder, and build the docker image:
+
+```bash
+$ docker build -t "akeneo-php" .
```
Comment (ahocquard) [inline php/README.md:26]: akeneo-fpm ?

File: fpm/README.md [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP FPM development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php). **This image does not contain Akeneo PIM**.
 
-It comes with PHP 7.1 (provided by dotdeb.org repository), and some PHP extensions: fpm of course,
-but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated,
-run `phpenmod xdebug` to enable it).
+It is intended to be used [httpd](https://hub.docker.com/_/httpd) with FCGI enable or [nginx](https://hub.docker.com/_/nginx).
 
-The PHP FPM daemon is configured with "docker" as user and group, and listen to the socket "/run/php/php7.1-fpm.sock".
+It comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: fpm of course, but also apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip, and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
```
Comment (ahocquard) [inline fpm/README.md:9]: `comes deactivated` seems weird but I'm not a very good english speaker :)

I would say:
`this last one is deactivated by default, run `phpenmod xdebug` to enable it`.

File: apache-php/README.md [hunk 8]
```diff
@@ -1,15 +1,12 @@
...
-This is a basic Docker environment for PHP development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php).
+[![Build Status](https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-7.1)](https://travis-ci.org/akeneo/Dockerfiles/tree/php-7.1)
 
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP development, based on [akeneo/php](https://hub.docker.com/r/akeneo/php). **This image does not contain Akeneo PIM**.
 
-It provides a pre-configured Apache (2.4) + mod_php (PHP 7.1) web server, based on Debian 8 (Jessie).
-Apache is configured to run with the `docker` user. It is then preferred to place your PHP application in `/home/docker`
-and modify the default VHOST to access it.
+It provides a pre-configured Apache (2.4) + mod_php ( PHP 7.1 from [Sury repository](https://deb.sury.org/)) web server, based on Debian 9 (Stretch).
```
Comment (rybus) [inline apache-php/README.md:7]: sometimes you write pre-configured while you also write it "preconfigured". looks like both are okay, though you have to be consistent.

File: fpm/README.md [hunk 9]
```diff
@@ -1,14 +1,14 @@
...
+
+The PHP FPM daemon is configured with "docker" as user and group, and istens to the port `9001`.
```
Comment (rybus) [inline fpm/README.md:11]: listens on port

File: php/README.md [hunk 10]
```diff
@@ -1,11 +1,36 @@
...
+[![Build Status](https://travis-ci.org/akeneo/Dockerfiles.svg?branch=php-7.1)](https://travis-ci.org/akeneo/Dockerfiles/tree/php-7.1)
 
-It is not intended to use directly as it stands, but rather to extend it and create custom development environments.
+This is a basic Docker environment for PHP development and testing, based on [debian:stretch-slim](https://hub.docker.com/_/debian/). **This image does not contain Akeneo PIM**.
 
-It comes with PHP 7.1 (provided by dotdeb.org repository), and some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd,
-mongo, soap, xml, zip and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+The environment comes with Debian 9 (Stretch) and PHP 7.1 (from [Sury repository](https://deb.sury.org/)), and some PHP extensions: apcu, mcrypt, intl, mysql, curl, gd, mongo, soap, xml, zip and xdebug (this last one comes deactivated, run `phpenmod xdebug` to enable it).
+
+## How to use it?
```
Comment (rybus) [inline php/README.md:9]: if you use "?" then it is a question, which must contain a verb. Otherwise, get rid of it ;)

File: php/README.md (hunk 6 above)
Comment (damien-carcel) [reply php/README.md:26]: No, this is the `akeneo/php` image

**Pull request merged**